app/component/AltTravelBar.js

```diff
@@ -59,4 +59,4 @@ class QuickSettingsPanel extends React.Component {
   }
 }
 
-export default QuickSettingsPanel;
+export default AltTravelBar;
```

Comment:
Does this actually contain anything else than the Datetime picker? If not, I would prefer to call this something related to the picker because this new name doesn't really tell me what this is any better than the old name. This file also uses a "quicksettings-container" class, maybe rename that too.

Reply:
True, this component actually rendered time picker and settings button. Renamed as SearchSettings instead of QuickSettings. Also renamed the styles,.